Code/Mantid/Framework/Kernel/src/VisibleWhenProperty.cpp

```diff
@@ -1,6 +1,3 @@
-#include "MantidKernel/VisibleWhenProperty.h"
-#include "MantidKernel/System.h"
```

Comment:
I bet you couldn't do this if you moved the implementation into the source file.